tests/testthat/test-util.R

```diff
@@ -6,3 +6,21 @@ test_that("null-or-value works", {
   expect_equal(NULL %||% NULL, NULL)
   expect_equal(NULL %||% 2, 2)
 })
+
+test_that("binrary serialisation is transitive", {
+  f <- function(x, identical = TRUE) {
+    y <- bin_to_object(object_to_bin(x))
+    is_equivalent_to <- if (identical) is_identical_to else equals
+    expect_equivalent(x, y)
```

Comment:
This is deprecated in new testthat in favour of `expect_equal(..., check.attributes = FALSE)` I think